Docs/Interaction_Logs.md

```diff
@@ -0,0 +1,42 @@
+## Interaction Log service
+### Background -
+The interaction log service will notify all the Firebolt calls getting invoked in the platform whether the Firebolt call is made manually by any action or internally made by any application or service. Whenever the platform identifies a firebolt call, the interaction log service will notify that.
+### Uses -
+The Interaction log service is essential for tracking method calls. When a method is called, it internally triggers/invokes other Firebolt methods. This service can be used to obtain the list of internal method calls.
+### Configurations -
+#### Framework side - fcs/config module side
+Below are the different ways to start the interaction service
+* Using `interactionsMetrics` flag. This can be enabled from the cli command to start the service
+* Using glue codes we can able to start and stop the interaction service and then using validation glue able to validate the obtained logs.
+* Using Before operation, able to start the service by adding `interactionsMetrics` to true in [beforeOperation](#before-operation). 
+
+#### Device side -
+Platform should have the support of interactonaction logging service. The way interactionlogs will be fetched from device differs on platform and that can be handled through config module.
+## Implementations -
```

Comment:
interactonaction  - this is a typo

Device side - I believe we should also mention about fb interaction here

Reply:
done